src/resources/views/ui/inc/scripts.blade.php

```diff
@@ -0,0 +1,82 @@
+@basset('https://unpkg.com/@popperjs/core@2.11.6/dist/umd/popper.min.js')
+@basset('https://unpkg.com/noty@3.2.0-beta-deprecated/lib/noty.min.js')
+@basset('https://unpkg.com/pace-js@1.2.4/pace.min.js')
+@basset('https://unpkg.com/sweetalert@2.1.2/dist/sweetalert.min.js')
+
+@if (backpack_theme_config('scripts') && count(backpack_theme_config('scripts')))
+    @foreach (backpack_theme_config('scripts') as $path)
+        @if(is_array($path))
+            @basset(...$path)
+        @else
+            @basset($path)
+        @endif
+    @endforeach
+@endif
+
+@if (backpack_theme_config('mix_scripts') && count(backpack_theme_config('mix_scripts')))
+    @foreach (backpack_theme_config('mix_scripts') as $path => $manifest)
+        <script type="text/javascript" src="{{ mix($path, $manifest) }}"></script>
+    @endforeach
+@endif
+
+@if (backpack_theme_config('vite_scripts') && count(backpack_theme_config('vite_scripts')))
+    @vite(backpack_theme_config('vite_scripts'))
+@endif
+
+@include(backpack_view('inc.alerts'))
+
+{{-- page script --}}
+<script type="text/javascript">
+    // To make Pace works on Ajax calls
+    $(document).ajaxStart(function() { Pace.restart(); });
+
+    // polyfill for `startsWith` from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith
+    if (!String.prototype.startsWith) {
+        Object.defineProperty(String.prototype, 'startsWith', {
+            value: function(search, rawPos) {
+                var pos = rawPos > 0 ? rawPos|0 : 0;
+                return this.substring(pos, pos + search.length) === search;
+            }
+        });
+    }
+
+
+
+    // polyfill for entries and keys from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries#polyfill
+    if (!Object.keys) Object.keys = function(o) {
+        if (o !== Object(o))
+            throw new TypeError('Object.keys called on a non-object');
+        var k=[],p;
+        for (p in o) if (Object.prototype.hasOwnProperty.call(o,p)) k.push(p);
+        return k;
+    }
+
+    if (!Object.entries) {
+        Object.entries = function( obj ){
+            var ownProps = Object.keys( obj ),
+                i = ownProps.length,
+                resArray = new Array(i); // preallocate the Array
+            while (i--)
+                resArray[i] = [ownProps[i], obj[ownProps[i]]];
+            return resArray;
+        };
+    }
+
+    // Ajax calls should always have the CSRF token attached to them, otherwise they won't work
+    $.ajaxSetup({
+        headers: {
+            'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
+        }
+    });
+
+    // Enable deep link to tab
+    document.querySelectorAll('.nav-tabs a').forEach(function (elem) {
+        if(elem.dataset.name === location.hash.substr(1)) (new bootstrap.Tab(elem)).show();
+        elem.addEventListener('click', () => location.hash = elem.dataset.name);
+    });
+</script>
```

Comment:
I think we can FINALLY move this to a CSS file. Maybe a proper name for it would be `common.js`? It's the same thing as we have for `common.css`, something that gets used by all themes, right?